webview/src/experiments/components/table/styles.module.scss

```diff
@@ -249,76 +353,12 @@ $bullet-size: calc(var(--design-unit) * 4px);
   }
 }
 
-ul {
-  padding-left: 1rem;
-}
-
-table {
-  display: inline-block;
-  border-collapse: collapse;
-}
-
-tr {
-  position: relative;
-  background-color: $row-bg-color;
-  width: fit-content;
-
-  &:hover.rowSelected {
-    &::after {
-      box-shadow: inset 0 0 20px $shadow;
-      content: '';
-      top: 0;
-      right: 0;
-      width: 100%;
-      height: 100%;
-      z-index: 4;
-      position: absolute;
-      pointer-events: none;
-    }
-
-    td:not(.experimentCell):hover::before,
-    td:hover + td::before {
-      background-color: $row-border-selected-color;
-    }
-  }
-
-  &:hover:not(.rowSelected) {
-    td:not(.experimentCell),
-    .experimentCell::before {
-      background-color: $row-hover-background-color;
-    }
-
-    td:not(.experimentCell):hover::before,
-    td:hover + td::before {
-      background-color: $border-color;
-    }
-  }
-
-  > *:first-child {
-    position: sticky;
-    left: 0;
-    z-index: 3;
-
-    &::after {
-      content: '';
-      height: 100%;
-      position: absolute;
-      top: 0;
-      width: 6px;
-      right: 0;
-      transition: box-shadow 0.25s;
-    }
-
-    &.headerCell {
-      &::after {
-        bottom: auto;
-        left: auto;
-        background-color: transparent;
-      }
+.bodyRow {
```

Comment:
The `tr {}` held styles that  only affect body rows (doesn't affect the head rows at all). I moved them outside of it and into the `bodyRow` selector. The body row styles can be cleaned up further but I'll save that for another chain pr that focused on cleaning up body styles.